snekbox/nsjail.py

```diff
@@ -205,8 +205,9 @@ def python3(
                 "--",
                 self.config.exec_bin.path,
                 *self.config.exec_bin.arg,
-                # Filter out empty strings (causes issues with python cli)
-                *(arg for arg in py_args if arg),
+                # Filter out empty strings at start of iterable
+                # (causes issues with python cli)
+                *(arg for i, arg in enumerate(py_args) if (arg or i > 0)),
```

Comment:
This doesn't cover the case of multiple empty strings at the start. I also think writing a test for this would be beneficial.

Reply:
Created function `iter_lstrip` for this task, and additional unit tests since [621c835](https://github.com/python-discord/snekbox/pull/159/commits/621c835f28cd3fcd1e5c3cccae9b5057647f9fa1)